packages/components/pie-modal/README.md

```diff
@@ -57,7 +57,7 @@ import '@justeattakeaway/pie-modal';
 ```js
 // React
 // For React, you will need to import our React-specific component build
-// which wraps the web component using @lit-labs/react
+// which wraps the web component using @lit/react
```

Comment:
I'm wondering whether we need this second line at all. Do consumers need to know that we use `@lit/react`?

Reply:
I am not sure, I would assume it was added for a reason.

Reply:
we agreed to address this in a follow-up PR as the react section of all components is misleading.